data/panel.html

```diff
@@ -84,7 +84,8 @@
   <div>
     <label><input type="radio" name="mode" id="countThreadHangs" /> Count thread hangs</label><br>
     <label><input type="radio" name="mode" id="countEventLoopLags" /> Count event loop lags</label>
-    <p class="description">Statuser can count either hangs on the main thread, or hangs on the event loop. Changing this will reset the hang counter.</p>
+    <label><input type="radio" name="mode" id="countInputEventResponseLags" /> Count input event response time lags</label>
+    <p class="description">Statuser can count either hangs on the main thread, lags on the event loop, or spikes in input event response time. Changing this setting will reset the hang counter.</p>
```

Comment:
In English, the word "either" should be used when describing an exclusive choice between two options. Sorta like if/else is the pair either/or. I recommend just dropping it.